plugins/woocommerce/tests/legacy/unit-tests/rest-api/Tests/Version3/system-status.php

```diff
@@ -44,12 +44,23 @@ public function setUp(): void {
 
 		wp_set_current_user( self::$user );
 
-		$this->endpoint = new WC_REST_System_Status_Controller();
-
 		// Callback used by WP_HTTP_TestCase to decide whether to perform HTTP requests or to provide a mocked response.
 		$this->http_responder = array( $this, 'mock_http_responses' );
 	}
 
+	/**
+	 * Fetches the system status data and caches it.
+	 */
+	private function get_system_status_data() {
+		static $system_status_data = null;
+		if ( ! $system_status_data ) {
+			wp_set_current_user( self::$user );
```

Comment:
Looking at this test class, it seems that the user is already being set in `setUp()` so we probably don't need this I think?